data/difficulty5_patched/gamestate/vehicle_types.xml

```diff
@@ -676,10 +676,10 @@
               <z>16</z>
             </size>
             <slices>
-              <entry>LOFTEMPS:xcom3/UFODATA/loftemps.DAT:xcom3/UFODATA/loftemps.TAB:122</entry>
-              <entry>LOFTEMPS:xcom3/UFODATA/loftemps.DAT:xcom3/UFODATA/loftemps.TAB:122</entry>
-              <entry>LOFTEMPS:xcom3/UFODATA/loftemps.DAT:xcom3/UFODATA/loftemps.TAB:122</entry>
-              <entry>LOFTEMPS:xcom3/UFODATA/loftemps.DAT:xcom3/UFODATA/loftemps.TAB:122</entry>
+              <entry>LOFTEMPS:xcom3/UFODATA/LOFTEMPS.DAT:xcom3/UFODATA/LOFTEMPS.TAB:122</entry>
```

Comment:
This seems to have changed, no idea why - regenerating this locally gives 'loftemps.DAT' like the previous commit?

Anyway, it makes sense to rewrite this commit without this, as it's nothing to do with this change and will pollute the history changing it back in another commit